src/hooks/web3Context.tsx

```diff
@@ -4,10 +4,19 @@ import WalletConnectProvider from "@walletconnect/web3-provider";
 import React, { ReactElement, useCallback, useContext, useMemo, useState } from "react";
 import { idFromHexString, initNetworkFunc } from "src/helpers/NetworkHelper";
 import { NodeHelper } from "src/helpers/NodeHelper";
+import WalletLink from "walletlink";
 import Web3Modal from "web3modal";
 
 import { NETWORKS } from "../constants";
 
+/**
+ *  Coinbase wallet init
+ */
+// const walletLink = new WalletLink({
+//   appName: "coinbase",
+// });
+// const walletLinkProvider = walletLink.makeWeb3Provider(NETWORKS[1].uri(), 1);
```

Comment:
I commented these out, because they didn't seem necessary & were causing type errors in `addTokenToWallet`...
I should go test that function real quick tho...

Reply:
yea wallet_watchAsset is not working with coinbase wallet. 
![image](https://user-images.githubusercontent.com/80423742/151397122-7f41942a-db64-4425-adc1-e6912318bf1a.png)

Reply:
What do you mean?